test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.AzureCodeGeneratorTest.test.ConsoleApp.csproj.approved.txt

```diff
@@ -7,6 +7,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.ML" Version="StablePackageVersion" />
     <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="StablePackageVersion" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.2.3" />
+    <PackageReference Include="Microsoft.ML.CodeGenerator.Helper" Version="1.2.3" />
```

Comment:
What is `Microsoft.ML.CodeGenerator.Helper`? Is the code for it included in this PR?

Reply:
It's a helper package which includes customer transformer,
See this issue for detail into
https://github.com/dotnet/machinelearning-modelbuilder/issues/397